stellar/README.md

```diff
@@ -100,6 +99,22 @@ node stellar/deploy-contract.js deploy interchain_token_service --chain-name <CH
 node stellar/deploy-contract.js deploy example --chain-name <CHAIN_NAME> --wasm-path ../axelar-cgp-soroban/target/wasm32-unknown-unknown/release/example.optimized.wasm
 ```
 
+### Contract upgrades
+To facilitate contract upgrades, the `upgrader` contract needs to be deployed first.
+
+```bash
+node stellar/deploy-contract.js deploy upgrader --chain-name <CHAIN_NAME> --wasm-path ../axelar-cgp-soroban/target/wasm32-unknown-unknown/release/upgrader.optimized.wasm
+```
+
+After the `upgrader` is deployed, any other instantiated contract can be upgraded by calling the `upgrade` function
+
+```bash
+node stellar/deploy-contract.js upgrade <CONTRACT_NAME> --chain-name <CHAIN_NAME> --wasm-path ../axelar-cgp-soroban/target/wasm32-unknown-unknown/release/<CONTRACT_NAME>.optimized.wasm --new-version <NEW_VERSION> --migration-data <MIGRATION_DATA>
+```
+
+where `<CONTRACT_NAME>` is the name of the contract to be upgraded and `--wasm-path` points to the upgraded bytecode. As a sanity check, `<NEW_VERSION>` must match the version number defined by the provided bytecode, so upgrading to the wrong version can be prevented. `<MIGRATION_DATA>` is the json encoded data that will be passed to the contract's `migrate` function.
```

Comment:
Can you mention how MIGRATION_DATA is encoded for the no migration case and a more complex migration case?